src/Microsoft.ML.CpuMath/CpuMathUtils.netcoreapp.cs

```diff
@@ -9,6 +9,174 @@ namespace Microsoft.ML.Runtime.Internal.CpuMath
 {
     public static partial class CpuMathUtils
     {
+        public static void MatTimesSrc(bool tran, bool add, AlignedArray mat, AlignedArray src, AlignedArray dst, int crun)
+        {
+            Contracts.Assert(mat.Size == dst.Size * src.Size);
+
+            if (Sse.IsSupported)
+            {
+                if (!tran)
+                {
+                    Contracts.Assert(0 <= crun && crun <= dst.Size);
+                    SseIntrinsics.MatMulA(add, mat, src, dst, crun, src.Size);
+                }
+                else
+                {
+                    Contracts.Assert(0 <= crun && crun <= src.Size);
+                    SseIntrinsics.MatMulTranA(add, mat, src, dst, dst.Size, crun);
+                }
+            }
+            else
```

Comment:
I just realized we don't have any unit tests for these software fallback implementations. I think that is OK for this PR, but it is something to put on your TODO list.